fetch-utils.js

```diff
@@ -27,3 +27,34 @@ export async function signOutUser() {
 }
 
 /* Data functions */
+
+
+export async function createItem(item) {
+    return await client.from('shopping_list').insert(item);
+
+}
+
+export function renderItem(item) {
+    const li = document.createElement('li');
+    li.textContent = item.quantity + ' ' + item.item;
+    return li;
+}
+
+export async function getListItems() {
+    return await client.from('shopping_list').select();
+
+}
+
+export async function gotIt(someId) {
+    const updatedItem = await client.from('shopping_list').update({ bought: true }).match({ id: someId });
+    return updatedItem;
+}
```

Comment:
you could just call it ```id``` instead of ```someId```, having ```id``` in the function parameter is basically saying "some ID"